src/transaction/combine.ts

```diff
@@ -17,26 +39,35 @@ function compareSigners(a, b) {
   )
 }
 
-function combine(signedTransactions: Array<string>): object {
-  validate.combine({signedTransactions})
-
-  // TODO: signedTransactions is an array of strings in the documentation, but
-  // tests and this code handle it as an array of objects. Fix!
-  const txs: any[] = _.map(signedTransactions, binary.decode)
-  const tx = _.omit(txs[0], 'Signers')
-  if (!_.every(txs, (_tx) => _.isEqual(tx, _.omit(_tx, 'Signers')))) {
-    throw new utils.common.errors.ValidationError(
-      'txJSON is not the same for all signedTransactions'
-    )
-  }
+function getATransactionWithAllSigners(transactions : Array<JsonObject>) : JsonObject {
   const unsortedSigners = _.reduce(
-    txs,
+    transactions,
     (accumulator, _tx) => accumulator.concat(_tx.Signers || []),
     []
   )
-  const signers = unsortedSigners.sort(compareSigners)
-  const signedTx = _.assign({}, tx, {Signers: signers})
-  const signedTransaction = binary.encode(signedTx)
+  //The documentation requires signers to be sorted
+  const sortedSigners = unsortedSigners.sort(compareSigners)
+  const tx = transactions[0];
+  tx.Signers = sortedSigners
```

Comment:
This is bad practice - this will modify the actual object at `transactions[0]`. That's why the syntax in deleted line 38 exists - so that a new object is created for that (I know it's a little weird). It may not matter for how you're using it, but someone else might use the same method and not realize.